apps/filecoin-site/src/app/(homepage)/page.tsx

```diff
@@ -197,18 +207,20 @@ export default async function Home() {
           title="Join the community"
           description="Be part of the movement to build a decentralized, efficient, and robust foundation for humanity's information."
         >
-          <CardGrid as="ul" cols="smTwo">
-            {communityLinks.map(({ title, href, icon }) => (
-              <LinkCard
-                key={title}
-                as="li"
-                backgroundVariant="dark"
-                title={title}
-                href={href}
-                icon={icon}
-              />
-            ))}
-          </CardGrid>
+          <CardGridContainer>
```

Comment:
If this is a one off, I prefer we don't abstract it. Makes it slightly more confusing about why it's a separate component. We could `CardGridContainers` that accept different max widths, but I wonder if it's necessary?